.github/workflows/partsrelationshipservice-deploy.yml

```
@@ -66,51 +66,61 @@ jobs:
       - name: "Build and push MTPDC images to ACR"
         run: |
           set -euo pipefail # safe mode
-          source cd/environments/${{ needs.configure.outputs.environment }}.tfvars
-          cd/build_and_push_docker.sh "$image_registry" "${{ matrix.image }}" "${{ needs.configure.outputs.image_tag }}"
+          source cd/environments/${{ needs.configure.outputs.environment }}.sh
+          cd/build_and_push_docker.sh "$TF_VAR_image_registry" "${{ matrix.image }}" "${{ needs.configure.outputs.image_tag }}"
+
+  deploy_common:
+    needs:
+    - configure
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: coreservices/partsrelationshipservice/cd/terraform-common
+    steps:
+      - uses: actions/checkout@v2
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+      - name: "Run Terraform, deploying infrastructure to Azure"
```

nit:  just to make it more clear for reader while looking at workflow run.
```suggestion
      - name: "Run Terraform, deploying common infrastructure to Azure"
```